supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchInitializationActor.scala

```diff
@@ -87,6 +95,64 @@ class AwsBatchInitializationActor(params: AwsBatchInitializationActorParams)
   private lazy val provider: Future[AwsCredentialsProvider] =
     Future { configuration.awsAuth.provider() }
 
+  lazy val secretsClient: SecretsManagerClient = {
+    val builder = SecretsManagerClient.builder()
+    configureClient(builder, Option(configuration.awsAuth), configuration.awsConfig.region)
+  }
+
+  private def storePrivateDockerToken(token: String) = {
+    try {
+
+      val secretName: String = "cromwell/credentials/dockerhub"
+
+      // Check if secret already exists
+      // If exists, update it otherwise create it
+      val secretsList: List[SecretListEntry] = secretsClient.listSecrets().secretList().asScala.toList
+      val secretsNameList = secretsList.map(_.name)
+
+      if(secretsNameList.contains(secretName)){
```

Comment:
Nit: I think you could simplify this slightly by using exists. So instead of defining `secretsNameList` and calling the contains method, you could make the condition `secretsList.exists(_.name == secretName)`.

Reply:
Done!